charms/argo-controller/test-requirements.txt

```diff
@@ -1,5 +1,5 @@
 # Copyright 2021 Canonical Ltd.
 # See LICENSE file for licensing details.
-juju<3.0
+
```

Comment:
I'm confused, did you remove the dependency instead of pinning it?

Reply:
Yes it will be installed anyway